testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -503,12 +506,15 @@ def params_from_base_test_setup(request, params_from_base_suite_setup):
 
     db = None
     cbl_db = None
+    test_db_log_file = None
+    path = None
     if create_db_per_test:
         if enable_file_logging and liteserv_version >= "2.5.0":
             cbllog = FileLogging(base_url)
             cbllog.configure(log_level="verbose", max_rotate_count=2,
-                             max_size=1000 * 512, plain_text=True)
-            log_info("Log files available at - {}".format(cbllog.get_directory()))
+                             max_size=100000 * 512, plain_text=True)
```

Comment:
same as above, it might effect log rotation tests

Reply:
same as above.